src/components/Header/Header.jsx

```diff
@@ -1,6 +1,12 @@
-// import the React library here
-// import a css file containig article styles
+import React from 'react';
+import './Header.css';
+import Navigation from '../Navigation/Navigation';
 
-// Create a Header function returning the HTML of header block
+const Header = () => (
```

Comment:
Also, you can use export here instead export default
```suggestion
export const Header = () => (
```

Reply:
Just For Your Information, here it's not working